tests/integration/test_fim/conftest.py

```diff
@@ -34,3 +37,26 @@ def create_key(request):
     file_monitor.start(timeout=SYNC_INTERVAL_VALUE + global_parameters.default_timeout,
                        callback=callback_detect_registry_integrity_clear_event,
                        error_message='Did not receive expected "integrity clear" event')
+
+
+@pytest.fixture(scope='module')
+def install_audit(get_configuration):
+    """Install auditd before test"""
+
+    # Check distro
+    linux_distro = id()
+
+    if re.match(linux_distro, "centos"):
+        package_management = "yum"
+        audit = "audit"
+        option = "--assumeyes"
+    elif re.match(linux_distro, "ubuntu") or re.match(linux_distro, "debian"):
+        package_management = "apt-get"
+        audit = "auditd"
+        option = "--yes"
+    else:
+        raise ValueError(f"Linux distro ({linux_distro}) not supported for uninstall/install audit")
+
+    # Install audit and start the service
+    process = subprocess.run([package_management, "install", audit, option], check=True)
+    process = subprocess.run(["service", "auditd", "start"], check=True)
```

Comment:
This was added to make install it probably because the machine being used did not have audit installed. It should be installed on the VM as part of our prerequisites instead of here.

There is a test that deals on removing and installing audit, but it's on the `test_audit/` suite.